src/TestLogger/Extensions/XunitTestAdapter.cs

```diff
@@ -46,6 +46,14 @@ public class XunitTestAdapter : ITestAdapter
                     result.Messages.Add(new TestResultMessage("skipReason", skipReason));
                 }
 
+                string displayName = result.Result.DisplayName;
+
+                // Add parameters for theories.
+                if (displayName is string dn && dn.Contains("("))
```

Comment:
`displayName` is already a string. Do we need the `is`?

Reply:
TransformShouldAddReasonForSkippedTests was creating entries where display name was null. I clarified the check though.